common/sgx/eeid.c

```diff
@@ -544,10 +574,23 @@ oe_result_t verify_eeid(
     // Check other image properties have not changed
     bool base_debug = sigstruct->attributes.flags & SGX_FLAGS_DEBUG;
     bool extended_debug = reported_attributes & OE_REPORT_ATTRIBUTES_DEBUG;
+    bool sgx_debug = reported_sgx_attributes->flags & SGX_FLAGS_DEBUG;
```

Comment:
Is `extended_debug` same as `sgx_debug`?

Reply:
Outdated. No, extended_debug is the debug flag that OE reports, not SGX.